head_test.go

```diff
@@ -465,7 +458,7 @@ func TestDeleteUntilCurMax(t *testing.T) {
 	numSamples := int64(10)
 	hb, err := NewHead(nil, nil, nil, 1000000)
 	testutil.Ok(t, err)
-	defer hb.Close()
+	defer testutil.Ok(t, hb.Close())
```

Comment:
As far as I know, the arguments of the function are evaluated beforehand where there is `defer`, so `hb.Close()` will get called here. This should do the trick:
```go
defer func() {
    	testutil.Ok(t, hb.Close())
}()
```

Reply:
yes you are right , I tested it in https://play.golang.org/p/1tq4gGY_FHm